helm/flowforge/templates/configmap.yaml

```
@@ -118,8 +118,9 @@ data:
         plausible:
           domain: {{ .Values.forge.telemetry.plausible }}
         {{ end -}}
-        {{ if .Values.forge.telemetry.posthog -}}
+        {{ if and (hasKey .Values.forge.telemetry "posthog") (hasKey .Values.forge.telemetry.posthog "apikey") -}}
         posthog:
+          apihost: {{ .Values.forge.telemetry.posthog.apihost }}
```

`apihost` isn't a required value - it can be omitted and the code uses the existing default.
```suggestion
          {{ if .Values.forge.telemetry.posthog.apihost -}}
          apihost: {{ .Values.forge.telemetry.posthog.apihost }}
          {{- end }}
```